how-to/access-isc-calendar.md

```diff
@@ -3,6 +3,16 @@ title: Access ISC Calendar
 type: docs
 prev: foundation-governance/how-to/
 ---
+# Adding our public and private calendars to your google calendar
+If you would like to be able to view the events in the private and public InnerSource Commons calendars you should add them to your calendar. This can be done by going to loging into your @innersourcecommons.org accoung and going to https://calendar.google.com/calendar/ and clicking this button: ![image](https://github.com/user-attachments/assets/3bac171f-caba-4f59-90a0-be06fce5f1fd) 
```

Comment:
```suggestion
If you would like to be able to view the events in the private and public InnerSource Commons calendars you should add them to your calendar. This can be done by logging into your @innersourcecommons.org account and going to https://calendar.google.com/calendar/ and clicking this button: ![image](https://github.com/user-attachments/assets/3bac171f-caba-4f59-90a0-be06fce5f1fd) 
```